### PR TITLE
dev-php/pecl-uuid: add 1.2.1

### PR DESCRIPTION
**Pull request by neutroniak**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: dev-php/pecl-uuid/pecl-uuid-1.2.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PHP_EXT_NAME="uuid"
+PHP_EXT_INIT="yes"
+PHP_EXT_ZENDEXT="no"
+DOCS=( CREDITS )
+
+USE_PHP="php8-2"
```
Comment (thesamesam) [inline dev-php/pecl-uuid/pecl-uuid-1.2.1.ebuild:11]: Could you test it with 8.3 too?

Comment (neutroniak) [reply dev-php/pecl-uuid/pecl-uuid-1.2.1.ebuild:11]: Hey, @thesamesam. 

Sure, I'll test it, too.

BTW, my pkgdev commit omitted a change on profiles/package.mask, to unmask pecl-uuid. Shall I add this change in the same PR?

Cheers,

Comment (thesamesam) [reply dev-php/pecl-uuid/pecl-uuid-1.2.1.ebuild:11]: Hi! Yes please, in a second commit for the unmask (same PR) after the one adding 1.2.1.

Comment (neutroniak) [reply dev-php/pecl-uuid/pecl-uuid-1.2.1.ebuild]: @thesamesam, works fine with php:8.3

$ php php.php 
version: 8.3.14
uuid_create: b29d5988-aca3-4457-8f1b-593ab2ad4a99

Please, check if the order of commits is fine.

Thanks

Comment (thesamesam) [reply dev-php/pecl-uuid/pecl-uuid-1.2.1.ebuild]: Looks good.

**Pull request closed without merging**